src/core/k8.ts

```diff
@@ -1190,6 +1205,72 @@ export class K8 {
     return resp.response.statusCode === 200.0
   }
 
+  /* ------------- ConfigMap ------------- */
+
+  /**
+   * @param name - name of the configmap
+   * @returns the configmap if found
+   * @throws SoloError - if the response if not found or the response is not OK
+   */
+  public async getNamespacedConfigMap (name: string): Promise<k8s.V1ConfigMap> {
+    const { response, body }  = await this.kubeClient.readNamespacedConfigMap(name, this._getNamespace())
+      .catch(e => e)
+
+    this._handleKubernetesClientError(response, body, 'Failed to get namespaced configmap')
+
+    return body as k8s.V1ConfigMap
+  }
+
+  /**
+   * @param name - for the config name
+   * @param labels - for the config metadata
+   * @param data - to contain in the config
+   */
+  public async createNamespacedConfigMap (name: string, labels: Record<string, string>, data: Record<string, string>): Promise<boolean> {
+    const namespace = this._getNamespace()
+
+    const configMap = new k8s.V1ConfigMap()
+    configMap.data = data
+
+    const metadata = new k8s.V1ObjectMeta()
+    metadata.name = name
+    metadata.namespace = namespace
+    metadata.labels = labels
+    configMap.metadata = metadata
+    try {
+      const resp  =await this.kubeClient.createNamespacedConfigMap(namespace, configMap)
+
+      return resp.response.statusCode === 201
+    } catch (e: Error | any) {
+      throw new SoloError(`failed to create configmap ${name} in namespace ${namespace}: ${e.message}, ${e?.body?.message}`, e)
+    }
+  }
+
+  /**
+   * @param name - for the config name
+   * @param labels - for the config metadata
+   * @param data - to contain in the config
+   */
+  public async replaceNamespacedConfigMap (name: string, labels: Record<string, string>, data: Record<string, string>): Promise<boolean> {
+    const namespace = this._getNamespace()
+
+    const configMap = new k8s.V1ConfigMap()
+    configMap.data = data
+
+    const metadata = new k8s.V1ObjectMeta()
+    metadata.name = name
+    metadata.namespace = namespace
+    metadata.labels = labels
+    configMap.metadata = metadata
+    try {
+      const resp  =await this.kubeClient.replaceNamespacedConfigMap(name, namespace, configMap)
```

Comment:
```suggestion
      const resp = await this.kubeClient.replaceNamespacedConfigMap(name, namespace, configMap)
```
I think you need to run `npm run format`